app/assets/stylesheets/layout.scss

```diff
@@ -313,6 +313,10 @@ table tbody td[align="right"], table tbody th[align="right"], table thead th[ali
   font-weight: 600;
 }
 
+#case-status-section .live-red-text {
+  padding-right: 10px;
```

Comment:
Looks like the font size is different to the lozenge, we should probably be consistent

Reply:
it's a bit of a weird one actually, if you put it to same font-size it somehow looks smaller. I think it's an optical illusion because of the lozenge background surround.

Reply:
Updated to be the same and it looks a little bit better I think

Reply:
Yeh I tried it too, strange